src/markers.js

```diff
@@ -8,8 +8,9 @@ const adjust_coordinates = (latlng, size) => [latlng.lat + size[1]/2, latlng.lng
 const EntityMarker = L.Marker.extend({
     update_size() {
         const tile_width = this.entity_map.latLngToLayerPoint([0,1]).x - this.entity_map.latLngToLayerPoint([0,0]).x
+        const padding = tile_width / 6  // want there to be small gap between icon and tile boundary
```

Comment:
Can you add the padding as an option in config.js instead of directly defining it in this function?